### PR TITLE
[MIG][10.0] Migrate openstreetmap module/snippet #4955

### PR DESCRIPTION
**Pull request by gfcapalbo**
Adds a website widget for openstreetmap.

### CODE REVIEW

**Comment by rafaelbn** [comment]
Hello @gfcapalbo ! Thanks for your contribution!

Please could you check travis errors? is :red_circle: 

@cristinamartinrod please take a look

Also could be very nice if you could review 3 PRs for each PR to make in order of maintenance :smile: just a suggestion :+1:

File: website_hr_extended/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```
Comment (nikul-serpentcs) [inline website_hr_extended/__openerp__.py:1]: @gfcapalbo Remove `__openerp__` file

File: website_openstreetmap/__init__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+##############################################################################
```
Comment (nikul-serpentcs) [inline website_openstreetmap/__init__.py:2]: @gfcapalbo Remove Copyright from all `__init__` file

File: website_openstreetmap/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    "depends": [
+        'website'
+    ],
+    "data": [
+        "views/view.xml",
+        "views/widget.xml",
+    ],
+    "auto_install": False,
+    "installable": True,
+    "external_dependencies": {
```
Comment (nikul-serpentcs) [inline website_openstreetmap/__manifest__.py:36]: @gfcapalbo IMO if not any `external_dependencies` then remove it.

File: website_openstreetmap/views/view.xml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
```
Comment (nikul-serpentcs) [inline website_openstreetmap/views/view.xml:2]: Replace with `odoo` and remove `<data>` tag.

File: website_openstreetmap/views/widget.xml [hunk 5]
```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
```
Comment (nikul-serpentcs) [inline website_openstreetmap/views/widget.xml:2]: same here

File: website_openstreetmap/views/widget.xml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    <data>
+
+        <!-- widget structure inferred from image floating widget it is placed in content and can be inside a text -->
+        <template id="openstreetmap_widget" inherit_id="website.snippets" name="openstreetmap">
+            <xpath expr="//div[@id='snippet_content']/div[@class='o_panel_body']" position="inside">
```
Comment (nikul-serpentcs) [inline website_openstreetmap/views/widget.xml:7]: here use `hasclass`

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Review by cristinamartinrod — CHANGES_REQUESTED** [review]
IMO is a bit difficult to adjust to any web design. The map always keeps 50% of the size of your block so the other half of the page is always empty. Also trying to obtain the corrections on company settings, shows me this error.
front website - map
error backend

**Pull request closed without merging**